CMakeLists.txt

```diff
@@ -94,17 +93,16 @@ else()
 
   option(WITH_BZ2 "build with bzip2" OFF)
   if(WITH_BZ2)
-    find_package(bzip2 REQUIRED)
+    find_package(BZip2 REQUIRED)
     add_definitions(-DBZIP2)
-    include_directories(${BZIP2_INCLUDE_DIR})
+    include_directories(${BZIP2_INCLUDE_DIRS})
```

Comment:
should we also update Findbzip2.cmake accordingly?

Reply:
`Findbzip2.cmake` is removed in the same commit.  please refer the commit message for more details.